2014-04-29 19:40:12,479 - [INFO] - from play in pool-4-thread-4 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2014-04-29 19:40:28,709 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 19:40:29,325 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:15:05,635 - [ERROR] - from application in New I/O worker #8 


! @6i41c61o9 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value signupForm]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-04-29 20:15:32,217 - [ERROR] - from application in New I/O worker #1 


! @6i41c7f8j - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value signupForm]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-04-29 20:15:57,115 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:15:57,370 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:17:47,995 - [ERROR] - from application in New I/O worker #8 


! @6i41cf851 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value signupForm]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-04-29 20:18:09,636 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:18:09,805 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:21:51,834 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:21:52,027 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:30:05,251 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:30:05,479 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:32:48,760 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:32:48,892 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:34:34,514 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:34:34,549 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:38:31,048 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:38:31,189 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:48:41,849 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:48:41,988 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:51:33,039 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:51:33,232 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:52:44,582 - [ERROR] - from application in New I/O worker #3 


! @6i41h4fgp - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-04-29 20:54:43,679 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:54:43,826 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:57:04,784 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:57:04,804 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:57:57,959 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:57:58,131 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 20:59:27,598 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 20:59:27,726 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:06:49,553 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:06:49,813 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:10:42,878 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:10:43,051 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:16:02,160 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:16:02,325 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:17:13,674 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:17:13,694 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:23:09,690 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:23:09,853 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:26:08,306 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:26:08,474 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:26:55,136 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:26:55,156 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:27:51,548 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:27:51,697 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 21:39:50,278 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 21:39:50,429 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:21:45,052 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:21:45,405 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:22:24,313 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:22:24,430 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:23:30,959 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:23:31,110 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:24:15,336 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:24:15,474 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:24:37,398 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:24:37,418 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:25:36,259 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:25:36,475 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:26:56,135 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:26:56,274 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:27:22,573 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:27:22,593 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:29:01,782 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:29:01,943 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:29:36,254 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:29:36,369 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:29:59,455 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:29:59,488 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:30:22,399 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:30:22,418 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:35:13,399 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:35:13,611 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:40:47,253 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:40:47,423 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:42:09,220 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:42:09,387 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:42:33,965 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:42:33,984 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:44:23,286 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:44:23,447 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:45:04,642 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:45:04,776 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-29 22:45:55,206 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-29 22:45:55,330 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-30 00:14:14,979 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:postgresql://localhost:5432/Diagram

2014-04-30 00:14:15,221 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-04-30 00:14:39,113 - [ERROR] - from play in play-internal-execution-context-1 
Cannot invoke the action, eventually got an error: javax.persistence.PersistenceException: The type [class models.userModel] is not a registered entity? If you don't explicitly list the entity classes to use Ebean will search for them in the classpath. If the entity is in a Jar check the ebean.search.jars property in ebean.properties file or check ServerConfig.addJar().

2014-04-30 00:14:39,304 - [ERROR] - from application in play-internal-execution-context-1 


! @6i42hhml1 - Internal server error, for (POST) [/login] ->

play.api.Application$$anon$1: Execution exception[[PersistenceException: The type [class models.userModel] is not a registered entity? If you don't explicitly list the entity classes to use Ebean will search for them in the classpath. If the entity is in a Jar check the ebean.search.jars property in ebean.properties file or check ServerConfig.addJar().]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.2]
	at play.api.DefaultApplication.handleError(Application.scala:399) ~[play_2.10.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) ~[play_2.10.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) ~[play_2.10.jar:2.2.2]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) ~[play_2.10.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) ~[play_2.10.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) ~[scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) ~[play-iteratees_2.10.jar:2.2.2]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) ~[scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) ~[scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) ~[scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]
Caused by: javax.persistence.PersistenceException: The type [class models.userModel] is not a registered entity? If you don't explicitly list the entity classes to use Ebean will search for them in the classpath. If the entity is in a Jar check the ebean.search.jars property in ebean.properties file or check ServerConfig.addJar().
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.createRequest(DefaultPersister.java:1281) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.saveRecurse(DefaultPersister.java:277) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:248) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1568) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1558) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebean.Ebean.save(Ebean.java:453) ~[avaje-ebeanorm.jar:na]
	at play.db.ebean.Model.save(Model.java:91) ~[play-java-ebean_2.10.jar:2.2.2]
	at controllers.Application.saveUser(Application.java:23) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:69) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$3$$anonfun$apply$3.apply(routes_routing.scala:69) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.2]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.2]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.2]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.2]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.2]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.2]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) ~[scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.2]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted

